packages/sku/lib/detectYarn.js

```diff
@@ -1,10 +0,0 @@
-const { execSync } = require('child_process');
-
-module.exports = () => {
-  try {
-    execSync('yarnpkg --version', { stdio: 'ignore' });
```

Comment:
`yarnpkg` is a rename that was done on debian machines I think. I think nowadays most people either just have `yarn`, or alias `yarn` to `yarnpkg`, so I'm calling the new logic package manager detection logic a "bugfix".

Reply:
Somehow, on my machine this command returns a version number, but in my terminal `yarnpkg` isn't a valid command.